### ISSUE TITLE
Viewing strikes

### ISSUE THREAD

**Issue by guipace** [flmorg/cleanuperr#27]
Is there a way to view the number of strikes a download has received?

I've configured the app, but can't tell if it's actually counting strikes towards removing the downloads. I also don't see any related info in the logs.

Thanks!

**Comment by Flaminel** [comment]
Enabling debug logging should start showing logs about the number of strikes an item has. By default, the logs show only when an item is removed due to receiving the max amount of strikes.

**Comment by guipace** [comment]
Forgot to mention I had already enabled debug logging. No strike info there.

**Comment by Flaminel** [comment]
Are there any items in the queue that should receive strikes? Are you using the latest docker image and enabled removing items with a certain number of strikes?

**Comment by oldretepzold** [comment]
![image](https://github.com/user-attachments/assets/a3cb6a67-afcc-4822-b03e-51d2247f926e)

Obviously not OP but can confirm strikes are logged.  However, I am curious why the current strike level is log level `debug` but everything else is `info`?

Could see it useful for some sort of CLI tool to get current strike info though.

**Comment by Flaminel** [comment]
No reason in particular. I just thought it would make sense for the number of strikes to be on `debug`, as they are not as important, and the actual removal and reason to be on `info`. Everything is up for debate though, so please tell me what you think makes more sense.

**Comment by Flaminel** [comment]
> Could see it useful for some sort of CLI tool to get current strike info though.

I did think of the possibility to add an API to cleanuperr, but it's not really a priority, especially if I can't find enough use cases for it. If you've got any more ideas, I'd be happy to receive any feature requests!

**Comment by oldretepzold** [comment]
> No reason in particular. I just thought it would make sense for the number of strikes to be on `debug`, as they are not as important, and the actual removal and reason to be on `info`. Everything is up for debate though, so please tell me what you think makes more sense.

Not a big deal, I would just assume they'd all be under info or all under debug.

**Comment by oldretepzold** [comment]
> > Could see it useful for some sort of CLI tool to get current strike info though.
> 
> I did think of the possibility to add an API to cleanuperr, but it's not really a priority, especially if I can't find enough use cases for it. If you've got any more ideas, I'd be happy to receive any feature requests!

Yeah not particularly high priority, just could be useful for a quick lookup vs looking through logs.

**Comment by guipace** [comment]
I can now view strikes. The issue was that I an on unraid and the application template used there was out of date.
Thanks @Flaminel for taking it over and providing a more correct one.

**Comment by Flaminel** [comment]
I'm glad it works!
